nxdrive/options.py

```diff
@@ -600,6 +604,14 @@ def _validate_exec_profile(value: str, /) -> str:
     raise ValueError("Can only be 'public' or 'private'.")
 
 
+def _callback_synchronization_enabled(new_value: bool) -> None:
+    log.warning(
+        """The option is deprecated since 5.1.2 and will be removed in a future release.
+         Use 'feature.synchronization' instead."""
```

Comment:
```suggestion
        "The option is deprecated since 5.1.2 and will be removed in a future release."
         " Use 'feature.synchronization' instead."
```